src/main/java/com/matera/cadastrocentral/authentication/PasswordDTO.java

```diff
@@ -0,0 +1,10 @@
+package com.matera.cadastrocentral.authentication;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class PasswordDTO {
+    private String newPassword;
```

Comment:
I think here would be better only `password` instead of `newPassword`. Because we can use this DTO in other requests that need the password, like for transfer and delete account.